src/c_kzg_4844.c

```diff
@@ -697,9 +706,6 @@ static C_KZG_RET blob_to_polynomial(Polynomial *p, const Blob *blob) {
     return C_KZG_OK;
 }
 
-/* Input size to the Fiat-Shamir challenge computation */
```

Comment:
I'm not sure how I feel about moving and grouping all these constants on the top. Especially for constants that are not globally relevant or globally needed, I think it makes sense to keep them close to the place where they are used. Both so that it's easier to see which part of the code they are used in, but also because IMO having them all grouped up on the top doesn't offer any major advantages to readability.

Reply:
No problem. I understand. I'll move `CHALLENGE_INPUT_SIZE` back to be above it's function.